NEWS.md

```diff
@@ -20,6 +20,7 @@ This release is in development and not yet ready for production use.
 
 ## Package
 
+- Fixed some typos in `single-timeseries-rt-estimation.Rmd`. The `WORDLIST` used by spelling has also been updated by eliminate false positives. Future typos will now generate an errors in continuous integration checks to catch them as early as possible. See #341 by @Bisaloo.
```

Comment:
```suggestion
- Fixed some typos in `single-timeseries-rt-estimation.Rmd`. The `WORDLIST` used by spelling has also been updated to eliminate false positives. Future typos will now generate an error in the continuous integration check. This will help us to catch them as early as possible. See #341 by @Bisaloo.
```